REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 1.0
 Distributions
-Distributed
\ No newline at end of file
+Distributed
+DataFrames
```

Comment:
We can start with this, but it would be nice if we could avoid a dependency on DataFrames.jl. Maybe we merge for now but add an issue that reminds us of this?

Reply:
I'm only using DataFrames because I thought it would be nice to return the results as a dataframe. Would it be better for the results to just be an array?